app/controllers/spotlight/dashboards_controller.rb

```diff
@@ -26,7 +26,7 @@ def load_recent_solr_documents count
       solr_params = { sort: "#{blacklight_config.index.timestamp_field} desc" }
       @response = query_solr({}, solr_params)
       @response.docs.take(count).map do |doc|
-        ::SolrDocument.new(doc, @response)
+        blacklight_config.solr_document_model.new(doc, @response)
```

Comment:
I probably started the bad habit of using `SolrDocument` directly, but we should pull the class name out of the blacklight config, in case our downstream consumers are doing stuff strangely.